documentation/src/utils/config.js

```diff
@@ -28,12 +28,16 @@ const docsConfig = {
     filesFromRepository: [
       {
         src: "scanners",
-        dst: "scanners",
+        dst: "docs/scanners",
       },
       {
         src: "hooks",
-        dst: "hooks",
+        dst: "docs/hooks",
       },
+      {
+        src: "resources/sponsors",
```

Comment:
Why that? We can simply reference the images IMO in the main README directly, right?